piecrust/src/store/session.rs

```diff
@@ -157,6 +158,41 @@ impl ContractSession {
             .map(|c| *c.index.root())
     }
 
+    /// Returns path to a file representing a given commit and page.
+    ///
+    /// Requires a contract's memory path and a main state path.
+    /// Progresses recursively via bases of commits.
+    pub fn do_find_page(
```

Comment:
isn't `find_page` a nicer name for this function?

Reply:
Ditto on `find_page`

Reply:
I like adding `do` to names :) as there are so many find_pages around, but I agree with your remarks - I will change that.